package.json

```diff
@@ -49,6 +49,7 @@
     "cssom": "^0.3.0",
     "domhandler": "^2.3.0",
     "escodegen": "^1.4.1",
+    "eslint": "^1.10.3",
```

Comment:
I think, in the long term, we need to find a better solution to include parsers / tools which also happen to be used for building astexplorer itself.